actioncable/app/assets/javascripts/action_cable/subscription.coffee

```diff
@@ -8,6 +8,12 @@
 #     connected: ->
 #       # Called once the subscription has been successfully completed
 #
+#     disconnected: (event) ->
```

Comment:
Yeah, maybe document it as `disconnected: ({ willAttemptReconnect: boolean }) ->`?